src/Stratis.Bitcoin.IntegrationTests.Common/EnvironmentMockUpHelpers/NodeBuilder.cs

```diff
@@ -86,10 +86,15 @@ public CoreNode CreateBitcoinCoreNode(string version = "0.13.1", bool useCookieA
 
         public CoreNode CreateStratisPowNode(bool start = false)
         {
-            return CreateNode(new StratisBitcoinPowRunner(this.GetNextDataFolderName()), start);
+            return CreateNode(new StratisBitcoinPowRunner(this.GetNextDataFolderName(), KnownNetworks.RegTest), start);
```

Comment:
@rowandh you know that beer you owe me? I'm calling in the favour ;)

Reply:
🍺 Done